src/System.IO/src/System/IO/MemoryStream.cs

```diff
@@ -474,41 +474,48 @@ public override int ReadByte()
             return _buffer[_position++];
         }
 
-
-        public override Task CopyToAsync(Stream destination, int bufferSize, CancellationToken cancellationToken)
+#if netcoreapp11
```

Comment:
I would expect this #ifdef to be needed. I'm not sure how this PR even compiles because this method is only virtual and thus can only be override in the netcoreapp1.1 version of System.Runtime which would need to be referenced here. However as far as I can tell it is only referencing the netstandard1.7 version of the System.Runtime which isn't virtual. I'm going to see if I can build this locally to see why it didn't fail.